tests/models/utils.py

```diff
@@ -188,13 +188,13 @@ def run_prediction(dataloader, trained_model, dp=False, min_acc=0.50):
         acc = torch.tensor(acc)
         acc = acc.item()
 
-    assert acc >= min_acc, f'this model is expected to get > {min_acc} in test set (it got {acc})'
+    assert acc >= min_acc, f"This model is expected to get > {min_acc} in test set (it got {acc})"
```

Comment:
we should be using single quotes  everywhere

Reply:
ok, I see it here https://www.python.org/dev/peps/pep-0498/ :]

Reply:
even I found both usage equally frequent... https://realpython.com/python-f-strings/